8.42.1

rhyspoulton opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->
This pull request updates to framel 8.42.1 which includes the frame spec 9 update.

Commit:
framel 8.42.1

Commit:
MNT: Re-rendered with conda-build 3.21.9, conda-smithy 3.20.0, and conda-forge-pinning 2022.06.06.01.05.55

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

Commit:
Add numpy to ignore_run_exports of python-framel

An error is ecountered due to a potential overdepending of numpy (because it contains SOs), but the numpy library is still required to use with python so it can be ignored.
